doc(release): Release notes collect 24.04.6 / gorgone 24.04.2 (#3591)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-os.mdx

## Centreon Collect

### 24.04.6

Release date: `September 25, 2024`

<details>
  <summary>Bug fixes</summary>

> - [Broker] when no filter is configured, forbidden filter is applied to all. 

Reviewer: ```suggestion
- [Broker] When no filter is configured on a Broker output, a default filter is applied so as to avoid Broker crashing.
```

> - [Engine] Fixed an issue regarding gRPC accepted services deletion.

Reviewer: ```suggestion
- [Engine] Engine now waits until no thread uses a gRPC service before deleting the service.
```

> - [Engine] When the Engine configuration is modified, no need to send host/service status since their configuration are already sent.

Reviewer: ```suggestion
- [Engine] When Engine's configuration is modified, already sent events are no longer sent a second time.
```

> - [Engine] When Engine was restarted, its log file wasn't reopened, so if it was moved or removed, it wasn't created again.

Reviewer: ```suggestion
- [Engine] Fixed the following issue: when Engine was restarted, its log file wasn't reopened, so if it was moved or removed, it wasn't created again.
```

## Centreon Gorgone

### 24.04.3

Release date: `September 25, 2024`

<details>
  <summary>Bug fixes</summary>

- [Gorgone] Added a partition for the Gorgone MBI mod_bi_metriccentileweeklyvalue and mod_bi_metriccentilemonthlyvalue tables, which are required by the ETL to run correctly.
- [Gorgone] Fixed an issue on cpu load when a poller was not responding.

> - [Gorgone] Fixed an issue where Gorgone running on the central host was leaking file descriptor when poller disconnect.

Reviewer: ```suggestion
- [Gorgone] Fixed an issue where gorgone running on a central server was leaking file descriptors when pollers were disconnected.
```
